### ISSUE TITLE
Creating droplet gives undefined as a response

### ISSUE THREAD

**Issue by phiresky** [pjpimentel/dots#3]
```ts
   doc.Droplet.create(specs).subscribe(
		obs => {
			console.log("obs", obs);
		},
		err => {
			console.log("err", err);
		}
	);
```

logs nothing and `let droplet = await doc.Droplet.create(specs).toPromise();` returns `undefined`.

The droplet is successfully created though and the response contains the correct data:

```
data: 
   { droplet: 
      { id: ...,
        name: ...,
        memory: 1024,
        vcpus: 1,
        disk: 25,
        locked: true,
        status: 'new',
        kernel: null,
        created_at: '2018-02-27T20:12:59Z',
...
```

**Comment by pjpimentel** [comment]
Hey, I will take a look at this soon! Thanks for reporting

**Comment by pjpimentel** [comment]
@phiresky I've just commited and published a new version 2.0.4 , could you try again and tell me if it is ok?

**Comment by phiresky** [comment]
Jep, seems to be working fine now. Thanks!
